docs/jwst/photom/main.rst

```diff
@@ -44,22 +45,30 @@ For spectroscopic data, if the photom step finds that the wavelength and relativ
 response arrays in the reference table row are populated, it loads those 1-D arrays
 and interpolates the response values into the 2-D space of the science image based
 on the wavelength at each pixel.
+For NIRSpec spectroscopic data and for NIRISS SOSS, the conversion terms in
+the photom reference file give results in flux density (MJy).  For point
+sources, the output of the photom step will be in these units.  For extended
+sources, however, the values will be divided by the average solid angle of a
```

Comment:
Might want to just mention that the source type determination comes from the value of the SRCTYPE keyword set earlier in the pipeline flow (and for NIRSpec MOS/SLIT is determined on a slit-by-slit basis).

Reply:
Thanks for your comments.  I'll fix all of these.